Assets/LeapMotion/Modules/HierarchyRecording/Scripts/LeapRecording/VectorHandRecording/VectorHand.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Recording {
+
+  public struct VectorHand {
+
+    public const int NUM_JOINT_POSITIONS = 25;
+
+    public bool       isLeft;
+    public Vector3    palmPos;
+    public Quaternion palmRot;
+    
+    private Vector3[]  _backingJointPositions;
+    public Vector3[] jointPositions {
+      get {
+        if (_backingJointPositions == null) {
```

Comment:
You could do `public readonly Vector3[] jointPositions = new Vector3[NUM_JOINT_POSITIONS]` and get the same functionality.  Can still read and write to the array, but not change it's size, and it's automatically allocated for you.

Reply:
Blarg I see, this is inside a struct, makes sense